src/components/Input/index.tsx

```diff
@@ -107,7 +107,7 @@ const Input = React.forwardRef<HTMLInputElement, InputProps>(
     const rightSideComponent = rightSideChild ?? <BsX strokeWidth={0.6} />
 
     return (
-      <div className={cn(formVariants({ variant }), formClassName)}>
+      <label className={cn(formVariants({ variant }), formClassName)}>
```

Comment:
Maybe I'm wrong, but I don't know if using a label here is the right approach.
What happen in a context where we compose a form using the Label component and the input component?

Normally, to create a relationship, a Label component could have an Input component as child, or use the `for` attribute to indicate the `id` of the Input

Maybe you can create a story to test this scenario.

Reply:
I did test all these scenarios before using it like this (tested that label around input works without for, tested with label wrapping the component and also tested with label alongside the component - with `htmlFor`) and it works in terms of getting the focus. It is true that it "breaks" the html specification...

Reply:
Yeah, my only concern is about a11y. But if you tested all the scenario is good! Thx